environment.yml

```diff
@@ -1,4 +1,4 @@
-name: base
+name: mne
```

Comment:
You're going to need to update the CIs to reflect this change, we assume `base` in our `azure-pipelines.yml` at least

https://github.com/mne-tools/mne-python/blob/master/azure-pipelines.yml#L150

Reply:
The particular line you're linking to is in the notebook/server CI job, so it actually should stay the way it is. But there are other CI changes needed; working on them now.